guide/command-handling/adding-features.md

```diff
@@ -31,6 +31,46 @@ In this short (but necessary) refactor, you:
 
 Now you can start adding features!
 
+## Command Categories
+
+So far all your command files are in a single `commands` folder. This is fine if you only have a few commands for your bot but if you're planning to make a bot that does lots of stuff you might end up with a large number of files in the `commands` folder. It will be a mess and thus a hard to maintain project. To fix this issue, you can sort your commands according to their types and put them in separate folders inside the `commands` folder. This sounds good, but it will throw an error if you try to do this with the current setup you have in the `index.js` to load commands. You will have to make a few changes to your existing code in order to make this all work out smoothly.
```

Comment:
-  "a bot... a bot" repetition
- "It will be a mess and thus a hard to maintain project" - that's very opinionated, would leave that out
- "To fix this issue" - technically not an issue
- "This sounds good, but it will throw an error if you try to do this with the current setup"  - "to make this change...", "for this to work"... etc. would be cleaner ways to coney the semantic meaning here